buildSrc/src/main/java/org/opensearch/gradle/zipplugin/ZipPublishUtil.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.gradle.zipplugin;
+
+class ZipPublishUtil {
+
+    static String capitalize(String str) {
+        return str.substring(0, 1).toUpperCase() + str.substring(1);
+    }
+
+}
```

Comment:
I think you call this with well known values, so just hard-code those values and remove this method.

Reply:
Util's I have added in consideration for future as well, we might to merge the existing publish task and this zippublish plugin to one common plugin, so then these handy Util's will come into picture. :)

Reply:
1. Publication name is a constant, `public final static String PUBLICATION_NAME = "mavenzip";`.
2. `MAVEN_ZIP_PUBLISH_POM_TASK` is a constant that uses that, `public final static String MAVEN_ZIP_PUBLISH_POM_TASK = "generatePomFileFor" + ZipPublishUtil.capitalize(PUBLICATION_NAME) + "Publication";`
3. So nothing is dynamic that needs to call `capitalize`, just do

```java
public final static String PUBLICATION_NAME = "mavenzip";
public final static String MAVEN_ZIP_PUBLISH_POM_TASK = "generatePomFileForMavenZipPublication";
```

Reply:
off course I can hardcode them, but if we change the `PUBLICATION_NAME`, then its expected from user to also modify `MAVEN_ZIP_PUBLISH_POM_TASK` to fix the publication, hence I added the Util.

Reply:
Why would we ever want to change these constants? I think you're over-optimizing for a complicated future problem by adding a new method (and no tests for it).